pom.xml

```diff
@@ -570,6 +570,12 @@
             </dependency>
 
             <dependency>
+                <groupId>joda-time</groupId>
+                <artifactId>joda-time</artifactId>
+                <version>2.9.9</version>
```

Comment:
Does airlift has any dependency on Joda? If so, maybe it should be bumped there first?